docs/specification.md

```diff
@@ -249,8 +251,8 @@ Each **Action** is described by the following attributes.
 
 Field | Description
 ----- | -----------
-`type` |  *string*. The type of action being performed. Allowed values are: `create`, `update`, `delete`. 
```

Comment:
Nit: Your changes should modify this line

Reply:
thx @kpshek.  I will wait to arrive at a consensus on this https://github.com/cds-hooks/docs/pull/112#issuecomment-347258271 and redo the PR